x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.test.tsx

```diff
@@ -547,7 +547,34 @@ describe('<CspPolicyTemplateForm />', () => {
     });
   });
 
+  describe('K8S', () => {
+    it('K8S or KSPM Vanilla should not render any Setup Access option', () => {
+      const policy = getMockPolicyK8s();
+
+      const { queryByLabelText } = render(<WrappedComponent newPolicy={policy} />);
+
+      expect(queryByLabelText('Assume role')).not.toBeInTheDocument();
```

Comment:
can we add test subjects to the fields and use them in the tests so they are less brittle?